pallets/identity/src/lib.rs

```diff
@@ -1645,20 +1641,21 @@ impl<T: Trait> Module<T> {
     ///     - You are not the owner of that CDD_ID.
     ///     - If claim is not valid.
     ///
-    fn base_add_investor_uniqueness_claim(
+    fn do_add_investor_uniqueness_claim(
```

Comment:
Not an improvement in naming imo, not sure what "do" adds since all functions are verbs and therefore do something. "base" conveys that its a helper function, which at least conveys some purpose & intent.